### PR TITLE
Feature/rdfa date docs

### PR DESCRIPTION
**Pull request by lagartoverde**
Tried to explain some symbols and give some examples, didn't want to overwhelm the user with text, we can put more or less if you think is better

### CODE REVIEW

**Review by elpoelma — CHANGES_REQUESTED** [review]
Looks good to me overall, I'll add dutch translations. Could you remove the inline css?

**Comment by elpoelma** [comment]
I also noticed that the Long year example is not correct.

**Comment by lagartoverde** [comment]
Done and done :)

**Review by elpoelma — APPROVED** [review]

File: translations/en-US.yaml [hunk 1]
```diff
@@ -63,6 +63,25 @@ besluit-plugin:
...
 date-plugin:
+  help:
+    first-line: "For custom formats, all the text between single quotes is escaped and written into the document as it's written, all the other characters are interpreted and represent different parts of the date:"
+    examples: "Examples:"
+    name: "Name"
+    symbol: "Symbol"
+    result: "Result"
+    detailed-info: For more detailed symbols visit
+    docs-link: the library documentation
+    date-examples: With this you can create any date format, some examples and it's corresponding date
```
Comment (abeforgit) [inline translations/en-US.yaml:74]: here are* some examples and their* corresponding dates*

File: translations/nl-BE.yaml [hunk 2]
```diff
@@ -63,10 +63,28 @@ besluit-plugin:
   placeholder:
     decision-title: Geef titel besluit op
 date-plugin:
+  help:
+    first-line: "Indien u een eigen datumnotatie wilt gebruiken, kunt u deze korte handleiding volgen. Datumnotaties kunnen bestaan uit zowel speciaal karakters/symbolen die verschillende delen van een datum voorstellen, en platte tekst die tussen aanhalingstekens ('') dient geplaatst te worden."
```
Comment (abeforgit) [inline translations/nl-BE.yaml:67]: speciale* karakters

File: addon/components/rdfa-date-plugin/card.hbs [hunk 3]
```diff
@@ -38,6 +38,7 @@
             />
           {{/if}}
         </AuFormRow>
+        <AuButton @icon="info-circle" {{on 'click' this.toggleHelpModal}}></AuButton>
```
Comment (abeforgit) [inline addon/components/rdfa-date-plugin/card.hbs:41]: I really don't like the look of this button
I'd suggest using the "link-secondary" skin and putting it next to the custom date control

**Review by abeforgit — CHANGES_REQUESTED** [review]

File: addon/components/rdfa-date-plugin/card.hbs (hunk 3 above)
Comment (elpoelma) [reply addon/components/rdfa-date-plugin/card.hbs:41]: An au-badge (https://appuniversum.github.io/ember-appuniversum/?path=/story/components-notifications-aubadge--component) is maybe also another possibility.

**Comment by abeforgit** [comment]
as mentioned in the chat, please link to the 2.28 version of date-fns

**Pull request merged**